tuneme/templates/comments/form.html

```diff
@@ -1,12 +1,17 @@
 {% load comments i18n wagtailcore_tags wagtailimages_tags %}
 
 
-<form action="{% comment_form_target %}" method="post">
+<form action="{% url 'molo-comments-post' %}" method="post">
     {% csrf_token %}
+    {% if form.errors %}
+      <h1>{% blocktrans count counter=form.errors|length %}Please correct the error below{% plural %}Please correct the
+        errors below{% endblocktrans %}</h1>
+    {% endif %}
 
+    {% if form.comment.errors %}{{ form.comment.errors }}{% endif %}
     <p>{{ form.comment }}</p>
     <input type="submit" name="submit" value="Comment">
-    {# <input type="submit" name="submit_anonymously" value="Add anonymously">#}
+    <input type="submit" name="submit_anonymously" value="Add anonymously">
```

Comment:
Not this is a "magic" flag recognised by molo.commenting, if that's present as a key in `request.POST` it'll use `Anonymous` as the name of the person submitting while still linking it to the user's profile.

Future releases should make `Anonymous` as a bit of text configurable.